### PR TITLE
Support for Wine

### PR DESCRIPTION
**Pull request by Peak-CDE**
I saw the other wine PR #66  and it is indeed broken. I based this version off of that one but actually made it work. This needs to be tested to make sure it doesn't affect windows functionality.

### CODE REVIEW

**Comment by CabbageMcGravel** [comment]
Quickly gave this a test.
The "Wine" option doesn't appear anywhere on my Windows machine, so that appears to be working!
I did a very simple export of a model without issues.

File: addon/types/model_export/model.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import bpy
 import shutil
 import subprocess
+import os
```
Comment (bonjorno7) [inline addon/types/model_export/model.py:4]: You'd bettter not be using os.path when pathlib exists.

File: addon/types/model_export/model.py [hunk 2]
```diff
@@ -12,6 +13,7 @@ class Model:
     def __init__(self, game, model):
         self.game = Path(game.game)
         self.bin = Path(game.bin)
+        self.wine = game.wine
```
Comment (bonjorno7) [inline addon/types/model_export/model.py:16]: Cast this to Path please.

**Review by bonjorno7 — COMMENTED** [review]
This is nice, but I would've liked to see you use `pathlib` instead of `os.path`.
I'm going to merge it into a branch, and do that there, to save you the trouble.

**Pull request merged**